app/views/user_registrations/link_github.html.slim

```diff
@@ -0,0 +1,19 @@
+.page-header
+  h3.text-center.green-text Welcome on board! Let's get started
+  
+  p.text-center During the program, you will need a Github repository and account to share your awesome code with your mentor.
+
+  p.text-center Don't have a GitHub account and repo? Check this guide on how to create one! 
+
```

Comment:
@anjali-sharma we just talked about it and we don't want to show github page at all during mentee registration. So mentee registration is just providing password and then we redirect her to her dashboard. this page won't be visible anywhere sorry